src/components/signTransaction.tsx

```diff
@@ -95,6 +100,67 @@ const SignTransaction = ({
     );
   }
 
+  const onSignTransactionRPC = async () => {
```

Comment:
Lot's of duplicate code with the other sign transaction function. Could we combine them with an arg in the function specifying whether to use RPC or not?

Reply:
Fixed, moved dupe code to new fn